README.md

```diff
@@ -1 +1,57 @@
-# Advanced-Building-Blocks---Bubble-Sort
\ No newline at end of file
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Advanced-Building-Blocks---Bubble-Sort
+
+> This is a collaborative project with Ruby for a Buble_sort.
+
+## Built With
+
+-Ruby
+
+## Instal
+- Download Ruby from https://www.ruby-lang.org/en/ to run it locally .
+
+
+## Clone
+- Clone the repository from https://github.com/pastorp3/Advanced-Building-Blocks---Bubble-Sort
+
+## Run
+-Run code with ruby main.rb in your local terminal
```

Comment:
- [ ] **OPTIONAL** You can add inline `code snippets` <-- by surrounding the code with backquotes (`)